pkg/madmin/heal-commands.go

```diff
@@ -494,10 +494,21 @@ func (adm *AdminClient) HealUpload(bucket, object, uploadID string, dryrun bool)
 
 // HealResult - represents result of heal-object admin API.
```

Comment:
This is on purpose the struct on cmd/ cannot be used with `madmin` package.. Just like how `minio-go` has ListObjectsV1 struct which is similar to what server has as ListObjectInfo.